README.md

```diff
@@ -96,6 +96,12 @@ Example ZLint CLI usage:
 
 	echo "Lint mycert.pem with all of the lints except for ETSI ESI sourced lints"
 	zlint -excludeSources=ETSI_ESI mycert.pem
+    
+    echo "Receive a copy of the full (default) configuration for all configurable lints"
+    zlint -exampleConfing
+
+    echo "Lint mycert.pem using a custom configuration for any configurable lints"
+    zlint -config configFile.toml mycert.pem
 
 See `zlint -h` for all available command line options.
 
```

Comment:
Do we need to update library usage to explain how to pass in a configuration? Do we also want someone to be able to programmatically provide configurations instead of requiring a toml based interface? It's a bit awkward to have a TOML based structure to this instead of TOML encoding an internal structure that the library expects. This could be solved by having Configuration{} object be initializable using data instead of toml.

Reply:
hmmm I admint that I completely forgot about using ZLint as a library.

I'll have to come back to this one, but at the end of the day it does all resolve down to Go structs. But honestly my answer to this depends on which layer the the library interface is exposed, which I'll get on top of.